tests/test_md.py

```diff
@@ -10,6 +10,8 @@
 
 from wfl.generate import md
 from wfl.configset import ConfigSet, OutputSpec
+from wfl.generate.md import select_every_10_fs_for_tests
```

Comment:
If these functions are for tests only, do they really belong in `wfl.generate.md`, or should they be moved into this file?

Reply:
the functions are only for tests, but I could not pickle them if they were defined in the same file. Any ideas how to get around that? If we can't work out how to define and pickle functions in the same file, I'll just move them to a file in the `tests` directory.

Reply:
Let me play with it a bit, and see if I can figure out how to do it cleanly.

Reply:
Your comment in the code suggests that you tried to define it as a fixture, and that failed. I think that's only true for `@` decorator wrapped functions (and that's why the autoparallelization wrapper is written the way it is).  But you can pickle a simple function that's defined in the same file.  The following patch works for me:
```
diff --git a/tests/test_md.py b/tests/test_md.py
index c2fa469..a647c4a 100644
--- a/tests/test_md.py
+++ b/tests/test_md.py
@@ -10,9 +10,11 @@ from ase.calculators.emt import EMT
 
 from wfl.generate import md
 from wfl.configset import ConfigSet, OutputSpec
-from wfl.generate.md import select_every_10_fs_for_tests
+# from wfl.generate.md import select_every_10_fs_for_tests
 from wfl.generate.md import check_validity_for_tests
 
+def select_every_10_fs_for_tests(traj):
+    return [at for at in traj if at.info["MD_time_fs"] % 10 == 0]
 
 @pytest.fixture
 def cu_slab():
diff --git a/wfl/generate/md.py b/wfl/generate/md.py
index 3487ea9..551b73e 100644
--- a/wfl/generate/md.py
+++ b/wfl/generate/md.py
@@ -258,12 +258,12 @@ def sample(*args, **kwargs):
 sample.__doc__ = autoparallelize_docstring(sample_autopara_wrappable.__doc__, "Atoms")
 
 
-# for tests, can't pickle if defined as a fixture
-def select_every_10_fs_for_tests(traj):
-    return [at for at in traj if at.info["MD_time_fs"] % 10 == 0]
+# # for tests, can't pickle if defined as a fixture
+# def select_every_10_fs_for_tests(traj):
+#     return [at for at in traj if at.info["MD_time_fs"] % 10 == 0]
 
 
 def check_validity_for_tests(at):
     if "5" in str(at.info["MD_time_fs"]):
         return False
-    return True
\ No newline at end of file
+    return True
```

Reply:
I thought I have tried this, but apparently not! Updated and seems to work.